improver_tests/psychrometric_calculations/wet_bulb_temperature/test_WetBulbTemperature.py

```diff
@@ -55,21 +53,6 @@ def setUp(self):
         self.latent_heat = np.array([2531771.0, 2508371.0, 2484971.0], dtype=np.float32)
         self.temperature = np.array([185.0, 260.65, 338.15], dtype=np.float32)
 
-    def test_calculate_latent_heat(self):
-        """Test latent heat calculation"""
-        expected = [2707271.0, 2530250.0, 2348900.0]
-        result = WetBulbTemperature()._calculate_latent_heat(self.temperature)
-        self.assertArrayAlmostEqual(result.data, expected, decimal=0)
-
-    def test_calculate_mixing_ratio(self):
-        """Test mixing ratio calculation"""
-        pressure = np.array([1.0e5, 9.9e4, 9.8e4], dtype=np.float32)
-        expected = [6.06744631e-08, 1.31079322e-03, 1.77063149e-01]
-        result = WetBulbTemperature()._calculate_mixing_ratio(
-            self.temperature, pressure
-        )
-        self.assertArrayAlmostEqual(result, expected, decimal=7)
-
```

Comment:
No problem with this section of code, but I was checking the rest of it had everything removed that needed to be (couldn't comment on that line). On line 94, the mixing ratio cube is set for the new Test_WetBulbTemperature class and I don't think it's ever used in that class.

Reply:
You're right. This is never used. I've removed it.